pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -91,6 +91,9 @@ func parseConfig(config *zconfig.EdgeDevConfig, getconfigCtx *getconfigContext,
 		parseNetworkInstanceConfig(config, getconfigCtx)
 		parseContentInfoConfig(getconfigCtx, config)
 		parseVolumeConfig(getconfigCtx, config)
+
+		// parseProfile must be called before processing of app instances from config
+		parseProfile(getconfigCtx, config, usingSaved)
```

Comment:
Where is this usingSaved coming from? The existing usingSaved has completely different semantics and purpose.